test/controllers/application_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Maybe I'm just confused but why is there an application controller test and an apply controller test.

Reply:
ApplicationController is default for the whole app, so had to name applying for bail as ApplyController